### DISCUSSION TITLE
Repository or organization Webhook?

### DISCUSSION THREAD

**Discussion by davidmrz** [GoogleCloudPlatform/fourkeys#250] [Q&A]
Hi, I'm not sure if this is the right way to ask a question but opening an issue didn't feel right.

I'm planning on integrating _fourkeys_ for our organization. I was wondering if the purpose of the tool is to be used on a single repository or does it make sense to integrate directly to the Github organization? (so that it captures all events on all repos).

Thanks!

**Comment by PiXeL16** [comment]
Had also the same though about the Org level. At the end what did you do @davidmr ?

**Reply by davidmrz** [reply]
I ended up using the Organization level webhook.

**Reply by PiXeL16** [reply]
Nice, did you end up also adding it as part of all your repos deployments as well?

**Reply by davidmrz** [reply]
No, I only added the organization one.
